cli/cook/subcommands/kill.py

```diff
@@ -48,51 +48,62 @@ def add(uuid, entity_type, cluster):
         raise Exception(message)
 
 
+def __kill_entities(cluster, uuids, endpoint, param, entity_type):
```

Comment:
Should this function return a dictionary with `success` and `failed` entries and let the printing be done elsewhere? This allows the caller to decide to group together the success and fail calls.

Reply:
I moved the printing outward, but this function still just returns `True`/`False` -- let me know what you think